### ISSUE TITLE
Value of type 'ApolloStore' has no member 'cacheKeyForObject'

### ISSUE THREAD

**Issue by indama** [apollographql/apollo-ios#2571]
## Bug report

Please replace this line with a short description of the problem. Make sure you've read `CHANGELOG.md` in the root of the repo to make sure a new version hasn't already addressed your problem!

## Versions

Please fill in the versions you're currently using: 

- `apollo-ios` SDK version: 1.0.0
- Xcode version: Final
- Swift version:  Final
## Steps to reproduce

Please replace this line with steps to reproduce the problem.

## Further details

Please replace this line with any further details or context necessary to understand the problem. Delete this section if you don't have anything further to add.

**Comment by indama** [comment]
Can you please make one demo/Example project with all settings and APIClient configurations? We are troubling with documentations. I think some of them outdated

**Comment by calvincestari** [comment]
> Can you please make one demo/Example project with all settings and APIClient configurations? 

Hi @indama - the tutorial will not be updated for 1.0 but we are working on getting the playground examples updated for 1.0 soon.

> Value of type 'ApolloStore' has no member 'cacheKeyForObject'

Configuration of cache keys has changed in v1.0, you can find the updated documentation [here](https://www.apollographql.com/docs/ios/caching/cache-key-resolution).

> We are troubling with documentations. I think some of them outdated

The documentation should be up-to-date but there is still more to be done to improve them based on the feedback we've received. Make sure you're looking at the [v1.0 documentation](https://www.apollographql.com/docs/ios/get-started) and not the older 0.x (legacy) documentation.

**Comment by calvincestari** [comment]
I'm going to close this issue as it's not a bug but a change in how cache key resolution now works.

**Comment by AnthonyMDev** [comment]
This needs to be part of the migration guide. We are going to be writing the migration guide ASAP. For now, the links in the docs Calvin mentioned are the best place to find this info.

**Comment by genkobar** [comment]
If anyone else, like me, ends up here because of the [Advanced networking configuration guide](https://www.apollographql.com/docs/ios/networking/request-pipeline/#example-interceptor-provider) and this code example:

```
...
struct NetworkInterceptorProvider: InterceptorProvider {
  ...

  func interceptors<Operation: GraphQLOperation>(for operation: Operation) -> [ApolloInterceptor] {
    return [
      MaxRetryInterceptor(),
      CacheReadInterceptor(store: self.store),
      UserManagementInterceptor(),
      RequestLoggingInterceptor(),
      NetworkFetchInterceptor(client: self.client),
      ResponseLoggingInterceptor(),
      ResponseCodeInterceptor(),
      JSONResponseParsingInterceptor(cacheKeyForObject: self.store.cacheKeyForObject), <------
      AutomaticPersistedQueryInterceptor(),
      CacheWriteInterceptor(store: self.store)
    ]
  }
}
```

It worked for me to just remove the cacheForObject parameter from the initializer for that interceptor:


```
      JSONResponseParsingInterceptor(), <------
```

**Comment by AnthonyMDev** [comment]
We should fix that documentation piece! Thanks for letting us know!

**Comment by genkobar** [comment]
@AnthonyMDev You're welcome! :)

**Comment by AnthonyMDev** [comment]
I've fixed this in the docs release for 1.1. Docs site will have the change published soon.
